setup.py

```diff
@@ -57,7 +57,7 @@
     'Flask>=0.11',
     'kubernetes>=6.0.0',
     'marshmallow>=2.13',
-    'reana-commons>=0.5.0.dev20190125,<0.6.0[kubernetes]',
+    'reana-commons>=0.5.0.dev20190212,<0.6.0[kubernetes]',
```

Comment:
Please squash and repush so that we don't publish addition and subsequent removal of `RUN pip install -e git://github.com/dinosk/reana-commons.git@154-FCC#egg=reana-commons` unnecessarily